extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -249,6 +251,15 @@ class UpdateReplicationStatus extends BackbeatTask {
                     replicationStatus:
                         updatedSourceEntry.getReplicationStatus(),
                 });
+                if (config.getIsTransientLocation(
+                    updatedSourceEntry.getDataStoreName()) &&
+                    updatedSourceEntry.getReplicationStatus()
+                    === 'COMPLETED') {
```

Comment:
shouldn't `getIsTransientLocation` take a location name arg?

Reply:
I think it does, confused by the weird indentation? :rofl:

Reply:
Ah my bad, I need glasses lol